ocs_ci/ocs/resources/mcg.py

```diff
@@ -968,6 +968,8 @@ def _compare_cli_hashes():
             assert (
                 _compare_cli_hashes()
             ), "Binary hash doesn't match the one on the operator pod"
+        logger.info("Retrieving mcg cli version")
+        self.exec_mcg_cmd("version")
```

Comment:
This version is very often not sufficient; It only changes once or twice across an entire version.

The info that's critical to log is the contents of `commit_hash`, in the root of the NB operator pod - which contains the exact version that can be later matched with a particular binary for debugging purposes